pipeline-steps/templates/steps/vault-upload.yaml

```diff
@@ -12,15 +12,15 @@ steps:
           scriptLocation: "inlineScript"
           inlineScript: |
             seachedSecret=$(az keyvault secret list --vault-name ${{ parameters.azureVault }}  --query "[?name=='${{ pair.secret_name }}'].name" -o tsv)
-            varValue='$(${{ pair.env_var_name }})'
+            var_value='$(${{ pair.env_var_name }})'
```

Comment:
Sorry, I think Qiang meant the pair.env_var_name to pair.var_value.
The original variable name varValue is fine.

Reply:
Remember to also change it to match in the mi-set-secrets.yaml

Reply:
made the changes